lib/vsc/install/commontest.py

```diff
@@ -64,6 +68,13 @@
     except ImportError:
         pass
 
+try:
+    from readme_renderer.rst import render
+    HAS_RENDERER = True
+except ImportError:
+    pass
+
```

Comment:
no need for all this trickery, 
just put `tests_require: ['readme_renderer']` in your setup.py